src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/diff/PrisonerDifferenceService.kt

```diff
@@ -60,20 +60,24 @@ class PrisonerDifferenceService(
       .filter { property -> property.findAnnotations<DiffableProperty>().isNotEmpty() }
       .associate { property -> property.name to property.findAnnotations<DiffableProperty>().first().type }
 
-  @Transactional
```

Comment:
No need for transaction to exist while diffs are being generated, hash has already been generated